modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java

```diff
@@ -574,14 +573,27 @@ private String _getStyleBookEntryTitle() {
 	}
 
 	private String _getThemeName() {
-		Group group = _themeDisplay.getScopeGroup();
+		long companyId = _themeDisplay.getCompanyId();
```

Comment:
These changes to the method need to be behind the feature flag. With the feature flag disabled we should still return the theme assigned to the public pages.